CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 2.1.0
```

Comment:
😉
```suggestion
## Version 2.2.0
```

Reply:
Ah, I fixed locally but forgot to force push 😅 thanks!